docs/components/dialogs/confirmDialog.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 0
+---
+
+# ConfirmDialog
+
+This is an overview of the most common usage of ConfirmDialog.
+
+## Live Demo
+
+<DocExampleBox codeBox="https://codesandbox.io/s/m7xmj8xnxj?module=%2Fsrc%2FApp.vue">
```

Comment:
you can remove the codex url and also add live demo prop.
< DocExampleBox :liveDemoMode="true" >